Probesys/bileto
# tec: Configure GreenMail

## Related issue(s)



https://github.com/Probesys/bileto/issues/330

## Changes



- add a new `mailserver` service to the docker-compose

## How to test manually



- start the dev environment with `make docker-start`
- follow the instructions from the documentation → check it works

## Checklist
